spring-boot-project/spring-boot-actuator/src/main/java/org/springframework/boot/actuate/management/PlainTextThreadDumpFormatter.java

```diff
@@ -82,7 +82,7 @@ private void writeStackTraceElement(PrintWriter writer, StackTraceElement elemen
 		writer.printf("\tat %s%n", element.toString());
 		LockInfo lockInfo = info.getLockInfo();
 		if (firstElement && lockInfo != null) {
-			if (element.getClassName().equals(Object.class.getName()) && element.getMethodName().equals("wait")) {
+			if (element.getClassName().equals(Object.class.getName()) && "wait".equals(element.getMethodName())) {
```

Comment:
`element.getMethodName()` does not return `null` so that change is not necessary.